core/src/main/resources/Samples/Input_Json_Specification.json

```diff
@@ -54,6 +54,8 @@
         "delimiter": "\t",
         "comment_mergeschema": "Optional, this is to merge different schemas when reading parquet files. the default value for this is false",
         "mergeschema": false,
+        "comment_coalescefilecount": "Optional, This will control the number of partitions to be written to the file system, hence those many files in the destination",
```

Comment:
Optional. Specify the number of partitions, which determine the number of files in the destination folder. Applies only to destinations, not sources.